src/realm/mixed.hpp

```diff
@@ -293,6 +293,7 @@ inline Mixed::Mixed(OldDateTime v) noexcept
 
 inline Mixed::Mixed(Timestamp v) noexcept
 {
+    REALM_ASSERT(false && "not yet implemented");
```

Comment:
This constructor is used by the object store, so introducing this assertion failure will break the bindings.